### ISSUE TITLE
where's the code corresponding to  "special_split_scheme"?

### ISSUE THREAD

**Issue by davidsonic** [KevinMusgrave/powerful-benchmarker#44]
Thank you for releasing powerful-benchmarkder! Can you specify where the code for the configuration option "special_split_scheme" is? I notice that the number reported in the 4-fold cross validation excel file does not perform worse than the one trained with original 50/50 train-test split, correct?

**Comment by KevinMusgrave** [comment]
Hey thanks for dropping by! So the code has gone through huge changes, and "special_split_scheme" no longer exists. And actually some of the uploaded config files (for the 4-fold experiments) aren't compatible with the latest code, so I have to go back and update them. Alternatively, you can install a previous version (I think 0.9.26 and pytorch-metric-learning == 0.9.86). The config files for the 50/50 spreadsheet are really old and there isn't a corresponding version number because I wasn't uploading to pypi at the time.

That said, I'm going to make sure everything is running smoothly before ECCV (next month) and I'll have a proper documentation webpage instead of just the readme.

Re: the 4-fold train/val/test performance being close to the 50/50 train/test performance, it's important to note that the 512-dim 4-fold accuracies are for an ensemble of 4 models, whereas the 512-dim 50/50 accuracies are for just 1 model.

**Comment by davidsonic** [comment]
Thank you for your reply! Yes, I notice that this repository has been constantly updated while I've been building on 0.9.26 and pytorch-metric-learning==0.9.86, for comparing with numbers in "A Metric Learning Reality Check" paper.  I assume changes in current repository don't affect results and conclusions in that paper?

For model ensemble, currently 4 128-dim split model is concatenated. Has the option of averaging over 4 512-dim split model been considered instead?

**Comment by KevinMusgrave** [comment]
Yeah the results/conclusions of the paper should still hold. The changes to the code have been for improving usability, modularity etc.

I haven't tested averaging over 4 512-dim models. It's an interesting idea though

**Comment by davidsonic** [comment]
Awesome! Issue closed!
